src/handlers/BaseHandler.js

```diff
@@ -15,7 +15,7 @@ class BaseHandler extends Component {
     const openApiUrl = this.props.location.query.url
     const parserType = this.props.parserType
     this.props.getDefinition(openApiUrl, parserType)
-    configureAnchors({ offset: 10 })
+    configureAnchors({ offset: -10, scrollDuration: 100 })
```

Comment:
I had to add `keepLastAnchorHash: true` to prevent the hash from being cleared each time I scrolled. Can you confirm if that happens for you too?